cmd/server/main.go

```diff
@@ -189,7 +190,18 @@ func realMain(ctx context.Context) error {
 		// API for creating new verification codes. Called via AJAX.
 		issueapiController := issueapi.New(ctx, config, db, h)
 		sub.Handle("/issue", issueapiController.HandleIssue()).Methods("POST")
-		sub.Handle("/checkcodestatus", issueapiController.HandleCheckCodeStatus()).Methods("POST")
+	}
+
+	{
+		sub := r.PathPrefix("/codestatus").Subrouter()
+		sub.Use(requireAuth)
+		sub.Use(requireRealm)
+		sub.Use(requireAdmin)
```

Comment:
should this require admin?

I don't actually have an answer to this.

Reply:
Actually no. We check later - nonadmins should be able to see codes but only the ones they've issued